deploy/infra/main.bicep

```diff
@@ -7,7 +7,7 @@ targetScope='subscription'
 param location string
 
 @minLength(3)
-@maxLength(8)
+@maxLength(15)
```

Comment:
I believe prefix is being used as part of the resource names. We need to test the resource deployment with max prefix length. Some resources are likely going to exceed their max name length.